components/NewRelicAgent/NRAgent.brs

```diff
@@ -265,7 +265,7 @@ function nrSendSystemEvent(actionName as String, attr = invalid) as Void
 end function
 
 function nrSendVideoEvent(actionName as String, attr = invalid) as Void
-    ev = nrCreateEvent("RokuVideo", actionName)
+    ev = nrCreateEvent("VideoAction", actionName)
```

Comment:
How are you differentiating `VideoAdAction`, `VideoErrorAction` & `VideoCustomAction`?